### ISSUE TITLE
Left Nav Release QA/508 Review

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#9567]
# Team Name
Facilities

# Product/Feature Name
VAMC template

## Checkpoint Stage
- [x] Other: Requesting 508 and QA review for defects identified as Release blockers during staging review of Left Navigation Menu 

## Point of contact
Michelle Middaugh

## Staging URL 
https://staging.va.gov/pittsburgh-health-care
No credentials needed 

## User stories for testing

[#9232](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9232)
- [X] Confirm presence of end to end tests for tab, click and verify focus (Gibran consulted with Trevor on these items so he will be an additional resource)

[#8961](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8961)
- [X] When a Veteran using a screenreader visits https://staging.va.gov/pittsburgh-health-care, the left navigation menu is presented contextually so that the Veteran can understand content elements as parent and child items within a list. 

[#8958](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8958)
- [X] When a Veteran views https://staging.va.gov/pittsburgh-health-care in narrow viewport or on mobile, the navigation is visible and not be cut off by other page components.

[#8975](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8975) 
- [ ] When a Veteran views https://staging.va.gov/pittsburgh-health-care in narrow viewport or on mobile, focus state should be visibly surrounding the relevant element on all sides. 
- [ ] When a Veteran views https://staging.va.gov/pittsburgh-health-care in narrow viewport or on mobile, focus state should be not be cut off by other page elements 
- [ ] When a Veteran views https://staging.va.gov/pittsburgh-health-care in narrow viewport or on mobile, focus state should be consistent with the design system.  

[#8974](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8974)
 - [X] When a Veteran views https://staging.va.gov/pittsburgh-health-care in narrow viewport or on mobile, focus state around the "In this Section" link should visibly surround the element on all sides. 

[#8906](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8906)
Active navigation elements not have the appearance or functionality of being available for selection once they are active and the page has been loaded. This should be tested for the following elements:
- [x] Health Services
- [x] Events
- [x] Stories
- [x] Policies
- [x] Contact Us

[#9162](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9162)
When viewing the left nav menu on https://staging.va.gov/pittsburgh-health-care
- [x] The white space should be visibly the same above, below and between all line items, whether the menu is open or closed.
- [x] Specifically, the padding above SERVICES AND LOCATIONS should be 22 pixels
- [x] Section Headings (ie: SERVICES AND LOCATIONS) should be rendered in 14
- [x] When a Veteran clicks on a parent page (e.g.Locations), the page should be loaded before the  the nav section becomes visible. 
- [x]  When a mobile user visiting https://staging.va.gov/pittsburgh-health-care clicks on any interactive element, scrolling is still available in the left nav.

## Next steps
- [ ] If the linked issue within the test case above is resolved, the QA or Accessibility specialist will close the related issue. 
- [ ] If the linked issue within the test case above remains unresolved, the QA or Accessibility specialist will comment in the related issue. 
- [ ] When all user stories above have been retested, the QA or Accessibility specialist will comment in this issue to confirm completion, tagging @mmiddaugh so that the team can coordinate any needed followup. 
- [ ] If additional work is needed, the Facilities team will coordinate another review after work is updated 
- [ ] Once all issues are addressed, the ticket is closed with a final comment, “All issues are addressed”, @-comment developer who opened issue, and close ticket/issue.
